src/MicroDB/Database.php

```diff
@@ -19,6 +19,11 @@ class Database
      * @var int
      */
     protected $mode;
+    
+    /**
+     * @var array
+     */
+    protected $options = array();
```

Comment:
Can you use the new 5.4 array syntax?
And also update composer.json to meet the new php 5.6 standard?